api/SpeechRecognition.json

```diff
@@ -35,10 +35,12 @@
             "version_added": false
           },
           "safari": {
-            "version_added": false
+            "prefix": "webkit",
```

Comment:
Prefixed-only support confirmed by https://mdn-bcd-collector.appspot.com/tests/api/webkitSpeechRecognition + https://mdn-bcd-collector.appspot.com/tests/api/SpeechRecognition.

The latter test didn't try using `webkitSpeechRecognition` if present, so I've fixed that in https://github.com/foolip/mdn-bcd-collector/pull/1130 and can confirm support for all members except `grammars` and `serviceURI`, which seems right.